Gemfile

```diff
@@ -1,3 +1,4 @@
 source "https://rubygems.org"
 
 gem "fastlane"
+gem 'cocoapods-no-autoimports'
```

Comment:
GitHub is complaining about a lack of a new line here.

Reply:
fixed [a00e1de](https://github.com/salemove/ios-sdk-widgets/pull/838/commits/a00e1de203c793d83b3c7bb06c8f106ef53bf89d)